packager/accept/accept.go

```diff
@@ -18,23 +18,57 @@ const SxgContentType = "application/signed-exchange;v=" + AcceptedSxgVersion
 // signedexchange library.
 var SxgVersion = version.Version1b3
 
+// Tokenize a comma-separated string of accept patterns into a slice
+func tokenize(accept string) []string {
+	var tokens []string
+	acceptLen := len(accept)
+	if acceptLen == 0 {
+		return tokens
+	}
+
+	inQuotes := false
+	startIndex := 0
+	for i := 0; i < acceptLen; i++ {
+		char := accept[i]
+		switch char {
+		case '"':
+			inQuotes = !inQuotes;
+		case ',':
+			if !inQuotes {
+				tokens = append(tokens, strings.TrimSpace(accept[startIndex:i]))
```

Comment:
[TrimSpace](https://golang.org/pkg/strings/#TrimSpace) trims too much. Use [TrimFunc](https://golang.org/pkg/strings/#TrimFunc) with a function that just checks for `' '` or `'\t'` (per the [#rule definition](https://tools.ietf.org/html/rfc7230#section-7)). Maybe define that function in [packager/util/http.go](https://github.com/ampproject/amppackager/blob/master/packager/util/http.go).

Ditto on line 45.

(Maybe this isn't a problem in practice? I'd prefer to be precise anyway.)

Reply:
This is addressed in a457c3273479f11f489be8ca02a07cb589eb3041